lighthouse-core/gather/gatherers/image-elements.js

```diff
@@ -359,14 +345,72 @@ class ImageElements extends Gatherer {
     if (reachedGatheringBudget) {
```

Comment:
I'm pretty sure this is bugged. `networkRecord` will always be true because `{}` is truthy.